sickchill/oldbeard/databases/music.py

```
@@ -0,0 +1,395 @@
+import datetime
+import logging
+import os
+from typing import List
+
+from slugify import slugify
+from sqlalchemy import ForeignKey, JSON
+from sqlalchemy.event import listen
+from sqlalchemy.ext.declarative import declarative_base
```

Ruff (0.12.2)

`sqlalchemy.ext.declarative.declarative_base` imported but unused

Remove unused import: `sqlalchemy.ext.declarative.declarative_base`

(F401)